src/main/java/week2/Node.java

```diff
@@ -0,0 +1,39 @@
+package week2;
```

Comment:
You're right, this can be a bit of a problem, if you've got things that you want to test but don't want to otherwise expose. You could add some helpers to the BTree to expose the Node functionality you want to test with. You could also make the Node class only visible to the package, and put the test class in the package. I'm not aware of a really elegant solution.